app/models/job_template.rb

```diff
@@ -72,8 +73,10 @@ def self.import(contents, options = {})
 
       template = existing || self.new
       template.sync_inputs(metadata.delete('template_inputs'))
-      template.assign_attributes(metadata.merge(:template => contents.gsub(/<%\#.+?.-?%>\n?/m, '')).merge(options))
+      template.sync_foreign_input_sets(metadata.delete('foreign_input_sets'))
+      template.assign_attributes(metadata.merge(:template => contents.gsub(/<%\#.+?.-?%>\n?/m, '')).merge(options).except('feature'))
       template.assign_taxonomies if template.new_record?
+      template.sync_feature(metadata.delete('feature'))
```

Comment:
I think you'll have to do the delete before `assign_attributes`. When we do the assign_attributes(metadata), feature will be in there but not an attribute of job_template.